stream-chat-android/src/main/java/com/getstream/sdk/chat/utils/StartStopBuffer.kt

```diff
@@ -27,7 +27,7 @@ internal class StartStopBuffer<T> {
     }
 
     private fun propagateData() {
-        GlobalScope.launch(DispatcherProvider.IO) {
+        CoroutineScope(DispatcherProvider.IO).launch {
```

Comment:
This change makes no difference, by the way. An uncancelled new scope might as well be `GlobalScope`. If anything, it uses more resources by creating the new scope here.